libraries/AP_Compass/AP_Compass.cpp

```diff
@@ -116,7 +116,8 @@ const AP_Param::GroupInfo Compass::var_info[] = {
     // @Description: Enable or disable the use of the compass (instead of the GPS) for determining heading
     // @Values: 0:Disabled,1:Enabled
     // @User: Advanced
-    // index 4
+
+`    // index 4
```

Comment:
is that back tick supposed to be there?

Reply:
well spotted, thanks.

Reply:
and I don't have those new glasses yet :)